XLS-0073-disallow-frozen-trustline-receiving/README.md

```diff
@@ -0,0 +1,92 @@
+<pre>
+Title:       <b>XLS-73d Disallow Receiving on Frozen Trustlines</b>
+Type:        <b>draft</b>
+Revision:    <b>1</b> (2024-07-29)
+Author:      <a href="mailto:shawnxie@ripple.com">Shawn Xie</a>
+Affiliation: <a href="https://ripple.com">Ripple</a>
+</pre>
+
+#  Disallow Receiving on Frozen Trustlines
+
+## Abstract
+
+This amendment empowers token issuers on the XRP Ledger to regulate tokens to prevent misuse by blacklisted accounts. The document outlines enhancements to the interactions between frozen assets and payments, ensuring that blacklisted token holders can no longer receive funds. These changes will significantly improve use cases such as stablecoins, where ensuring regulatory compliance is essential.
```

Comment:
I'd suggest adding a bit more on the regulation front about why this change is necessary and why the status quo doesn't follow most regulation frameworks.

Reply:
Maybe suggest a different paragraph? In my proposed change, I think it's important to _not_ say that token issuers regulate their own tokens because regulator and issuer are distinct.

Reply:
I don't have a specific idea of what to say, because I don't understand the regulation myself (and why this makes such a big difference). And if _I_ don't really understand the regulation, then most other people probably won't either.

Reply:
> I don't have a specific idea of what to say

That sounds fine. I don't have anything more specific to say either, so maybe let's just keep the spec focused on technical concepts? What's there now accomplishes that from my perspective.

Reply:
My concern was mostly about people (especially UNL validators) going "why do we need this change?" and I'm not sure this abstract does a complete job of addressing that question.